Resources/views/CRUD/base_list_field.html.twig

```diff
@@ -11,14 +11,12 @@ file that was distributed with this source code.
 
 <td class="sonata-ba-list-field sonata-ba-list-field-{{ field_description.type }}" objectId="{{ admin.id(object) }}"{% if field_description.options.row_align is defined %} style="text-align:{{ field_description.options.row_align }}"{% endif %}>
     {% set route = field_description.options.route.name|default(null) %}
-    {% set action = route == 'show' ? 'VIEW' : route|upper %}
 
     {% if
         field_description.options.identifier is defined
         and route
-        and action
         and admin.hasRoute(route)
-        and admin.isGranted(action, action in ['VIEW', 'EDIT'] ? object : null)
+        and admin.hasAccess(route, action in ['show', 'edit'] ? object : null)
```

Comment:
I get it @ZRazor , this action should be route.... can you create a PR? otherwise I will do it tonight